packages/patternfly-4/react-core/src/helpers/GenerateId/GenerateId.d.ts

```diff
@@ -5,6 +5,6 @@ export interface GenerateIdProps {
   children(id: string): ReactNode;
 }
 
-declare const GenerateId: ComponentClass<GenerateId>;
+declare const GenerateId: ComponentClass<GenerateIdProps>;
```

Comment:
this was a miss on my part - did not see the error in my IDE for some reason, but only see it in the built dist downstream...